npm-shrinkwrap.json

```diff
@@ -5,72 +5,61 @@
     "abbrev": {
       "version": "1.1.0",
       "from": "abbrev@>=1.0.0 <2.0.0",
-      "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.0.tgz",
-      "dev": true
```

Comment:
What npm version are you running? We recently encountered a similar problem when running an older npm version. I'm on 3.10.10, which seems to work fine.